common/trinary/trit_byte.c

```diff
@@ -38,29 +38,37 @@ byte_t trits_to_byte(trit_t const *const trits, byte_t const cum,
   return trits_to_byte(trits, cum * RADIX + trits[i], i - 1);
 }
 
+void _byte_to_trits(byte_t byte, trit_t *const trits, size_t const j, size_t const i) {
```

Comment:
Do we really need last parameter `i`?
If `bytes_to_trits` is called with *n_trits == 1* doesn't it mean that remaining trits are all 0 anyways?
If so, you can always do: `trits[i] = trit;` and get same results, make sense?

Reply:
Yes, because we have to start converting with the highest order index (j) but only get the trits for the lower order. Potentially, i can be less than j on initial call (it happens in flex_trit)